test/ComposerRequireCheckerTest/FileLocator/LocateAllFilesByExtensionTest.php

```diff
@@ -96,6 +97,7 @@ public function testLocateWithDirectoryBlackList(): void
      *
      * @dataProvider provideBlacklists
      */
+    #[DataProvider('provideBlacklists')]
```

Comment:
Do we need both of these comments? Can we remove the docblock tag in favour of the attribute?

Reply:
https://github.com/maglnet/ComposerRequireChecker/pull/564#discussion_r1946739239